### DISCUSSION TITLE
Can I set a custom alias for a store path so that data can be obtained using the alias?

### DISCUSSION THREAD

**Discussion by Peterpan-Coder** [zarr-developers/zarr-python#1283] [Q&A] [answered]
Can I set a custom alias for a store path so that data can be obtained using the alias?

**Comment by joshmoore** [comment, marked as answer]
@Peterpan-Coder: not that I know of. For filesystem-base stores, you might be able to achieve something similar with a symlink.

**Comment by Peterpan-Coder** [comment]
Thank you very much for your reply, which provides me with a good idea !


------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "zarr-developers/zarr-python"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2022年12月7日(星期三) 下午4:49
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [zarr-developers/zarr-python] Can I set a custom alias for a store path so that data can be obtained using the alias? (Discussion #1283)





 
@Peterpan-Coder: not that I know of. For filesystem-base stores, you might be able to achieve something similar with a symlink.
